spec/features/visitors/sign_up_spec.rb

```diff
@@ -82,24 +89,8 @@
       expect(current_path).to eq users_otp_path
     end
 
-    it 'disables OTP lockout during account creation' do
```

Comment:
We don't currently have any lockout for phone confirmation, or expiry, or attempt count even.

The phone confirmation will expire with the user's session which i think it reasonable.

I'm not totally sure we want an attempt counter?  What is the threat model here?  An attacker can create an account and link it to a phone they don't control?  Anything else?  I guess this would currently prevent you from registering your phone with another account?   I'd like to re-visit this as we discuss phone number re-use.  There has been some discussion recently about not forcing OTP phone numbers to be unique.